### ISSUE TITLE
shape argument

### ISSUE THREAD

**Issue by masalmon** [dill/emoGG#4]
I think it would be great to have an argument "shape" similar to the one in geom_point so that the emoji varies according to the level of a factor variable. Then one would also need to provide a "scale" of emoji (for having this level of the factor = this emoji). Easier said than done, obviously. For now I obtain the same result using add_emoji in a loop.

**Comment by dill** [comment]
Agreed, this would be nice! In the meantime, does this solve your problem?

https://gist.github.com/dill/ef79cdd21bf3a1977641

**Comment by masalmon** [comment]
Oh yes it does! Many thanks! The only problem with this code is that emojis do not have a nice shape: if I make the graph wider (increasing the size of the graphics pane in RStudio) then they are too wide compared to their height. But it really is a nice piece of code, thanks again. :-) (github does not have cool emojis ;-) )

**Comment by dill** [comment]
I'll try to take a look at the scale issue soon and put something 
together and report back to this thread when I do.

Thanks for the suggestion!

**Comment by zx8754** [comment]
Related SO post: http://stackoverflow.com/q/40355540/680068

**Comment by zx8754** [comment]
Solution provided in the gist doesn't work anymore, throws an error:

> Error: Not Found

**Comment by pboesu** [comment]
for the record: subsetting can be used as a workaround. I've updated @dill's gist here: https://gist.github.com/pboesu/9527bc8e4facbc6f863710af4a440653 (can't do a PR on a gist, unfortunately)
